Doc/whatsnew/3.13.rst

```diff
@@ -115,6 +115,18 @@ array
   It can be used instead of ``'u'`` type code, which is deprecated.
   (Contributed by Inada Naoki in :gh:`80480`.)
 
+copy
+----
+
+* Add :func:`copy.replace` function which allows to create a modified copy of
+  an object, which is especially usefule for immutable objects.
```

Comment:
usefule -> useful

Reply:
Thanks. Do you mind to create a PR to fix this typo?